src/main.c

```diff
@@ -219,6 +221,9 @@ core_opt(struct agent_core_t *core, int argc, char **argv)
 		case 't':
 			core->config->timeout = strtod(optarg, NULL);
 			break;
+		case 'w':
```

Comment:
I agree that `w` is reasonable, but I always feel anxious when I select a new letter for an option like this. There is no `-w` in `varnishd`, so I say go for it.

Reply:
Yeah, choosing that letter was the hardest part of the PR, I stared for quite a while at the keyboard.